app/jobs/open_order_cycle_job.rb

```diff
@@ -20,6 +19,8 @@ def perform(order_cycle_id)
 
       # And notify any subscribers
       OrderCycles::WebhookService.create_webhook_job(order_cycle, 'order_cycle.opened', opened_at)
+    rescue ActiveRecord::RecordNotFound
+      # do nothing
```

Comment:
This shouldn't really happen, should it? I'm wondering if we should report this to Bugsnag. Otherwise there may be a case when we are wondering: Why did the job not do its job? And we won't have any record of that.

I think that this only happens if the syncing takes so long that five minutes passed and the job gets scheduled again.

Reply:
You're right, we shouldn't suppress the error.

Because Sidekiq retries jobs that raise an exception (for about 20 days), I chose to suppress the error. 
I also don't want to set `sidekiq_options retries: 0`, in case there's some other error, and maybe it should be retried (although maybe only for 10 mins). Sidekiq doesn't seem to have a way to configure retries based on a condition.

So.. I'm thinking to set `sidekiq_options retry_for: 10.minutes` and not have any error handling. What do you think?